SU-HSDO/suhumsci
# Overflow hero

# READY FOR REVIEW

# Summary
- Overflow the main container to make the hero image full width.
- might need some tweaking.
- move the css to where you need.

# Needed By (Date)
- whenver

# Urgency
- low

# Steps to Test

1. Checkout this branch
2. `blt sync:db --site=archaeology`
3. add a hero image to the bottom of the home page
4. ensure it goes full width.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
